invenio_swh/models.py

```diff
@@ -52,7 +52,7 @@ class SWHDepositModel(db.Model, Timestamp):
     swhid = db.Column(db.String(1024), nullable=True)
     """Software Hash ID."""
 
-    swh_deposit_id = db.Column(db.String, nullable=True)
+    swh_deposit_id = db.Column(db.String, nullable=True)  # TODO add an index
```

Comment:
```suggestion
    swh_deposit_id = db.Column(db.String, nullable=True)
```

Let's add it :)

It should also be a unique index since we can't archive the same thing twice (and if it happens it would be nice to catch at the DB level even)